src/actions/search.js

```diff
@@ -40,3 +40,8 @@ export const removeDateRange = () => ({
 export const resetDateRange = () => ({
   type: types.RESET_DATE_RANGE,
 });
+
+export const changeSortMethod = sortMethodValue => ({
```

Comment:
`changeSortMethod` is not a good function name.  Because it is not clear and has two verbs in this name. You can name it as `sortSearchResultsBy`, then from the function name, I will know what responsibilities it will take. Also, you can rename the argument `sortMethodValue` to `type`. We always use short term to name a local variable in a function scope.

Reply:
Finished